AcquisitionInfo/explainer.md

```diff
@@ -0,0 +1,275 @@
+acquisition details through the `getDetails()` method which returns a ScriptPromise object.
+
+```js
+details = await navigator.acquisitionInfoProvider.getDetails();
+let installSource = details["installSource"]; // e.g. Microsoft Store
+let campaignId = details["campaignId"]; // e.g. adCampaign202306
+```
+
+While there are some common key-value pairs, depending on the platform of the acquisition the
+pairs contained within the dictionary may differ to fit different use cases.
+
+> Payload examples are taken from [Play Install Referrer][Play Install Referrer API] and [Apple Ads Attribution][Apple Ads Attribution API]
+documentation and are not final.
+
+```js
+details = {
+  installSource: "Google Play Store",
+  installReferrer: "utm_source=google-play&utm_medium=organic",
+  referrerClickTimestampSeconds: 1623214800,
+  installBeginTimestampSeconds: 1623214810,
+  installBeginReferrer: "utm_source=google-play&utm_medium=referral&utm_campaign=campaign-123",
+  installVersion: "1.0.0",
+  googlePlayInstant: false
+}
+```
+
+```js
+details = {
+  installSource: "Apple App Store",
+  attribution: "true",
+  orgId: "40669820",
+  campaignId: "542370539",
+  conversionType: "Download",
+  adGroupId: "542317095",
+  countryOrRegion: "US",
+  keywordId: "87675432",
+  adId: "542317136"
+}
+```
+
+In the event that a web application has referral information that has expired for that specific
+application, or in any other case where acquisition info is missing for the web application,
+the API would return empty values for the associated properties in the dictionary. This is consistent with
+existing behavior in store information retrieval API for native applications.
+
+```js
+details = {
+  installSource: "Microsoft Store",
+  campaignId: ""
+}
+```
+
+## Attribution on browser installs
+
+While attribution is currently not being tracked in the same capacity as Store installations for browser-initiated
+installations of web applications, the Acquisition Info API could enable this sort of attribution capability.
+
+### Same-domain installation
+
+Currently the main method of installing a PWA through the browser is navigating to the desired web application and
+initiating a same-domain installation. It may be the case that the intial navigation was due to an ad click.
+By having the user agent (UA) capture a stadardized property (`attributionId`) for what led the user to the page
+upon installation, it becomes possible for the Acquistion Info API to return attribution information for browser-installed
+web applications as well.
+
+#### Example same-domain use case
+
+1. User clicks on a Bing ads campaign that leads them to _foo.com?atr\_id=bingAdsAug2023_.
+
+2. User installs the _foo.com_ PWA from _foo.com_.
+
+3. UA captures the attribution information at the time of installation.
+
+4. Calling the Acquisition Info API `getDetails()` for _foo.com_ PWA produces the following payload.
+
+```js
+details = {
+  installSource: "foo.com",
```

Comment:
In this document, you never explicitly cover the expectations for how `installSource` is expected to work.  You have examples, but you're never explicit.

Somewhere in here, you should cover all of the possible scenarios:

Possible values for `installsource`:

1. `Policy` - If the web app was installed due to an enterprise policy
2. `Microsoft Store` | `Google Play Store` | `Apple App Store` - If the web app was installed through one of these stores
3. `<install source origin>` - For same-site installation, the web app's domain name, for cross-domain installation, the installer's domain name.

Did I miss any?